content/pt-br/docs/reference/node/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Informações de referência do Node
+weight: 80
+no_list: true
+---
+
+Esta seção contém os seguintes tópicos de referência sobre os nodes:
+
+* O [checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) do kubelet
```

Comment:
How do you feel about using "A API de checkpoint do kubelet" instead?

"checkpoint API" is not very idiomatic in Portuguese.